### PR TITLE
Update named instruments list

### PR DESCRIPTION
**Pull request by Wrymouth**
That's all I got right now, this code is confusing...
Instruments should be consistently named/numbered now, with no duplicates.

### CODE REVIEW

**Review by nanaian — APPROVED** [review]
Need to merge this into web

**Comment by Wrymouth** [comment]
Just completed the JSON instrument list, though there's a couple instruments where I'm not sure what category they belong in, categorized as "Unsure" for now.

File: mamar-web/src/app/instruments.ts [hunk 1]
```diff
@@ -61,9 +176,28 @@ export const categories = [
...
         ],
     },
-    // TODO: onwards from 0x03,0x2D..
+    {
+        name: "Unsure",
+        instruments: [
+            // are accordions keyboards?
+            { bank: 0x03, patch: 0x89, name: "Accordion 1" },
+            { bank: 0x03, patch: 0x8A, name: "Accordion 2" },
+            { bank: 0x03, patch: 0x8B, name: "Accordion 3" },
```
Comment (nanaian) [inline mamar-web/src/app/instruments.ts:194]: Yes, put them there please!

File: mamar-web/src/app/instruments.ts [hunk 2]
```diff
@@ -61,9 +176,28 @@ export const categories = [
...
+            { bank: 0x03, patch: 0x4C, name: "Steel Drum 3" },
+            { bank: 0x03, patch: 0x72, name: "Tubular Bells" },
+            { bank: 0x03, patch: 0xA2, name: "Plucked Percussion (?)" },
+            { bank: 0x03, patch: 0xA4, name: "Reverse Cymbal" },
+            { bank: 0x03, patch: 0xE4, name: "Woodblock" },
         ],
     },
-    // TODO: onwards from 0x03,0x2D..
+    {
+        name: "Unsure",
```
Comment (nanaian) [inline mamar-web/src/app/instruments.ts:189]: Can you name this "other"?

File: mamar-web/src/app/instruments.ts [hunk 3]
```diff
@@ -61,9 +176,28 @@ export const categories = [
...
+        instruments: [
+            // are accordions keyboards?
+            { bank: 0x03, patch: 0x89, name: "Accordion 1" },
+            { bank: 0x03, patch: 0x8A, name: "Accordion 2" },
+            { bank: 0x03, patch: 0x8B, name: "Accordion 3" },
+
+            // these are just animal sounds
+            { bank: 0x03, patch: 0x98, name: "Mosquito" },
+            { bank: 0x03, patch: 0x99, name: "Cat [Lead 8 (bass + lead)]" },
```
Comment (nanaian) [inline mamar-web/src/app/instruments.ts:198]: Remove the part in square brackets; just call it cat meow or something

**Review by nanaian — CHANGES_REQUESTED** [review]

**Pull request merged**